web/views.py

```
@@ -909,14 +921,15 @@ def github_update(request):
+    """Send a message to the configured Slack webhook if available."""
+    webhook_url = getattr(settings, "SLACK_WEBHOOK_URL", "")
+    if not webhook_url or "://" not in webhook_url:
+        print("Warning: SLACK_WEBHOOK_URL not configured or invalid")
         return
 
     payload = {"text": f"```{message}```"}
     try:
-        response = requests.post(webhook_url, json=payload)
+        response = requests.post(webhook_url, json=payload, timeout=5)  # Added timeout
         response.raise_for_status()  # Raise exception for bad status codes
     except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)